SPECS/openblas/openblas.spec

```diff
@@ -194,12 +189,10 @@ This package contains the static libraries.
 tar zxf %{SOURCE0}
 cd OpenBLAS-%{version}
 %if %{with system_lapack}
-%patch0 -p1 -b .system_lapack
+%patch 0 -p1 -b .system_lapack
 %endif
-%patch1 -p1 -b .libname
-%patch3 -p1 -b .sbgem
-%patch4 -p1 -b .tests
-%patch5 -p1
+%patch 1 -p1 -b .libname
```

Comment:
Are we intentionally skipping patch2?

Reply:
After digging the change history a bit, looks like patch2 was imported from Fedora while importing the openblas package. Fedora uses it for RHEL, so it is skipped here.
https://src.fedoraproject.org/rpms/openblas/blob/rawhide/f/openblas.spec#_242